surveys/forms.py

```diff
@@ -165,14 +165,34 @@ def __init__(self, *args, **kwargs):
         self.fields['study'].required = True
         self.fields['location'].required = True
 
+        self.fields['name'].help_text = "Survey names should be unique and memorable."
+        self.fields['type'].help_text = "This selection will determine which questions \
+                                        are available to add to your survey."
+        self.fields['survey_template'].help_text = "Choose a template if you'd like to \
+                                                   prepopulate your survey with a set of \
+                                                   recommended questions. You'll still be  \
+                                                   able to edit them on the next screen."
+
     class Meta:
         model = survey_models.SurveyFormEntry
-        fields = ['user', 'name', 'study', 'location', 'type', 'active']
+        fields = ['user', 'name', 'study', 'location', 'type', 'survey_template', 'active']
         widgets = {
             'user': forms.HiddenInput(),
             'active': forms.HiddenInput(),
         }
 
+    def clean_survey_template(self):
+        template = self.cleaned_data['survey_template']
+        survey_type = self.cleaned_data['type']
+
+        if template and (template.surveyformentry.type != survey_type):
+            if survey_type == 'intercept':
+                raise forms.ValidationError("Make sure you choose an intercept template to create a new intercept survey!")
+            if survey_type == 'observational':
+                raise forms.ValidationError("Make sure you choose an observational template to create a new observational survey!")
```

Comment:
We could simplify this a little bit to generalize to both cases:

```python
if template and (template.surveyformentry.type != survey_type):
    raise forms.ValidationError(
        "Make sure you choose an {} template to create a new {} survey!".format(survey_type)
    )
```

Reply:
Oo true!

Reply:
Needed a slight syntax change:

```python
if template and (template.surveyformentry.type != survey_type):
	raise forms.ValidationError(
	"Make sure you choose an {0} template to create a new {0} survey!".format(survey_type)
	)
```